src/whitelist/sites.conf

```
@@ -160,6 +160,7 @@ api-inference.huggingface.co
 data.firebaseio.com
 data.mongodb-api.com
 r2.cloudflarestorage.com
+mysql.db.mdbgo.com
```

Thanks! for completeness:

```suggestion
mongo.db.mdbgo.com
mysql.db.mdbgo.com
```

After that i'll merge this.